api/pkg/apis/v1alpha1/managers/devices/devices-manager.go

```diff
@@ -29,10 +32,12 @@ type DevicesManager struct {
 func (s *DevicesManager) Init(context *contexts.VendorContext, config managers.ManagerConfig, providers map[string]providers.IProvider) error {
 	err := s.Manager.Init(context, config, providers)
 	if err != nil {
+		log.Errorf("M (Devices): failed to initialize manager %+v", err)
```

Comment:
Sorry I haven't mentioned this - we use one space before manager-level logs and two spaces before provider-level logs. This allows us to get indentations in the log views. This might be helpful to understand the calling chains in some cases. What do you think?
![image](https://github.com/eclipse-symphony/symphony/assets/19833345/01c538c4-ae28-4f48-87a4-dfa744d96d3f)

Reply:
Good to known.